### ISSUE TITLE
Incomplete translation

### ISSUE THREAD

**Issue by jairPineros** [cityofaustin/techstack#4614]
## Describe the bug
The word fields (in english) still appears on the translation of the page.
## URL
https://alpha.austin.gov/es/municipal-court/financial-affidavit-form/
## To Reproduce
Steps to reproduce the behavior 
example: 
1. Go to https://alpha.austin.gov/es/municipal-court/financial-affidavit-form/
2. Select one of the any option under "INFORMACIÓN FINANCIERA / DECLARACIÓN JURADA DE INDIGENCIA"
3. Click next
4. In the next page **do not** fill any field and click next

## Current behavior
there is an English word in between a block of text on Spanish 

## Expected behavior
to prevent confusion when users make a mistake the warning and instructions text should not contain both english and spanish 
## Screenshots

![Screen Shot 2020-07-09 at 2.28.58 PM.png](https://images.zenhubusercontent.com/5f05cb2ac67f7f6ab3c0b762/6ab3e96c-980a-4748-b850-15a9be95a3c2)
## Device and browser

Apple computer /Google chrome

**Comment by gabrielaperezdelaosa** [comment]
Thanks for catching this @jairPineros , this is actually an issue in Formstack (this page is a form container). @InaraFernandez is e-mailing their customer service about it.

**Comment by InaraFernandez** [comment]
From formstack: "Unfortunately, at this time we do not have an option to customize the error messages, but this is really a great idea and maybe something that we will add in the future. I'd recommend submitting this as a feature request." 

I've submitted a feature request and will follow-up with our rep in a few weeks in case I don't hear back. I did notice in a list of related feature requests that other people have reported similar errors.
